org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -30,6 +30,9 @@ <h3>New Features</h3>
   <li>Add parameter to include the current project in the <code>report-aggregate</code>
       Maven goal
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/1007">#1007</a>).</li>
+  <li>Component accessors generated by the Java compilers for records are filtered
+      out during generation of report. Contributed by Tesla Zhang‮
```

Comment:
On this line GitHub shows the following warning

> This file contains bidirectional Unicode text that may be interpreted or compiled differently than what appears below. To review, open the file in an editor that reveals hidden Unicode characters. [Learn more about bidirectional Unicode characters](https://github.co/hiddenchars)

because there is `U+202E` at the end of the line.

And this file in Chrome looks as following

<img width="974" alt="Screenshot 2023-02-05 at 01 38 07" src="https://user-images.githubusercontent.com/138671/216795559-25443acf-7ecc-40bc-a312-c3e51e25ee52.png">

So I believe that this character should be removed.

Reply:
Done

Reply:
My bad. Thanks for spotting and fixing it!